doc/conf.py

```diff
@@ -105,16 +105,18 @@ def _warn_node(self, msg, node):
 
 # General information about the project.
 project = u'Luigi'
-copyright = u'2014, Erik Bernhardsson and Elias Freider'
+copyright = u'2015, Erik Bernhardsson and Elias Freider'
```

Comment:
@econchick yes, I agree with you. This is maybe the first commit that mentioned luigi.

```txt
SHA:	999630bd4043c0c12e096c9e930d42ba76d4907d
Author:	Erik Bernhardsson <erikbern@spotify.com>
Date:	Wed Nov 23 2011 07:55:31 GMT+0800 (CST)
Subject:	renamed builder3 to luigi in the code
Parent:	07dee441cac1f9bcf28851a08087fbc9e00b360a
```

`year` variable is not defined, and it seems there's no way to get the closest year.

Maybe getting it dynamically is a good way. e.g.

```python
copyright = u"2011-{}, Erik Bernhardsson and Elias Freider".format(datetime.datetime.now().year)
```